manticore/__main__.py

```diff
@@ -1,10 +1,17 @@
 import sys
+import logging
 import argparse
 
-from manticore import Manticore
+from . import Manticore
+from .utils import log
```

Comment:
(mostly commenting to doc tech debt, not necessarily requiring a fix. i understand this is the first pass to get it to work)

ideally we shouldn't access internal modules in `__main__.py`, and only call `init_logging` once (i believe its called three times between `__main__.py:main`, `__main__.py:ethereum_cli`, and the top of `manticore.py`)

Reply:
Agree on ideally. I updated `init_logging()` to keep initialization in one place, to make porting easier once we make a more generic `main()`